paasta_tools/kubernetes_tools.py

```diff
@@ -1325,7 +1325,7 @@ def get_boto_volume(self) -> Optional[V1Volume]:
         secret_name = limit_size_with_hash(f"paasta-boto-key-{service_name}")
         volume = V1Volume(
             name=self.get_sanitised_volume_name(
-                f"secret-boto-key-{service_name}", length_limit=253
+                f"secret-boto-key-{service_name}", length_limit=63
```

Comment:
might be nice to add a function like
```Python
def get_boto_secret_volume_name(self, service_name: str) -> str:
    return self.get_sanitised_volume_name(
        f"secret-boto-key--{service_name}", length_limit=63
    )
```
alongside all the other `get_*_volume_name` functions so that we only have one place to modify the volume name format and length